src/node.cc

```diff
@@ -847,6 +861,13 @@ void PromiseRejectCallback(PromiseRejectMessage message) {
 
   if (event == v8::kPromiseRejectWithNoHandler) {
     callback = env->promise_reject_unhandled_function();
+
+    Persistent<Promise>* p = new Persistent<Promise>(env->isolate(), promise);
```

Comment:
I can’t seem to find where this object is deleted :/

Reply:
I am unsure what has to be done here. I know to little C++ :-/